src/util/PopulationsMap.test.ts

```diff
@@ -2694,10 +2694,139 @@ it("add defaults observations for boolean and Patient based testcases when loadi
 
   expect(testCase.populationValues.length).toBe(3);
 
-  const updatedTestCase = mapExistingTestCase(testCase, ratioGroup);
+  const updatedTestCase = mapExistingTestCasePopulations(testCase, ratioGroup);
   expect(updatedTestCase.populationValues.length).toBe(5);
   expect(updatedTestCase.populationValues[2].name).toBe(
     "denominatorObservation"
   );
   expect(updatedTestCase.populationValues[4].name).toBe("numeratorObservation");
 });
+
+it("add defaults observations for Episode based testcases when loading the test case", () => {
+  const testCase = {
+    groupId: "654a77facda7fe554f6ba799",
+    scoring: "Ratio",
+    populationBasis: "false",
+    populationValues: [
+      {
+        id: "65dc5880-0161-476a-a3eb-888ddf47c636",
+        criteriaReference: null,
+        name: "initialPopulation",
+        expected: "1",
+        actual: null,
+      },
+      {
+        id: "c1af4146-b8a0-4e33-8eac-eba5daa07d2d",
+        criteriaReference: null,
+        name: "denominator",
+        expected: "6",
+        actual: null,
+      },
+      {
+        id: "c826a547-d457-47f1-a6cb-13b46ded0f48",
+        criteriaReference: null,
+        name: "denominatorExclusion",
+        expected: "2",
+        actual: null,
+      },
+      {
+        id: "b6c3659b-4dfa-4f07-864b-44562e0d2b2a",
+        criteriaReference: null,
+        name: "numerator",
+        expected: "4",
+        actual: null,
+      },
+      {
+        id: "numeratorObservation0",
+        criteriaReference: "b6c3659b-4dfa-4f07-864b-44562e0d2b2a",
+        name: "numeratorObservation",
+        expected: 0,
+        actual: null,
+      },
+      {
+        id: "061eec61-a363-4f9e-9999-0f62fc8a615f",
+        criteriaReference: null,
+        name: "numeratorExclusion",
+        expected: "3",
+        actual: null,
+      },
+    ],
+    stratificationValues: [],
+  };
+
+  const ratioGroup = {
```

Comment:
I see a TypeScript error, to avoid it can we Type this object to "Group" ?

Reply:
Sure, I'll make that change